README.md

```diff
@@ -29,3 +46,16 @@ const customTheme = createMuiTheme({
     <SchemaViewer />
 </ThemeProvider>
 ```
+
+## Contributing
+This repository uses Neutrino and `neutrino-preset-react-components` for developing, previewing, and building React components. To get started:
+- Fork and clone this repo.
+- Install the dependencies with `yarn`.
+- Start the development server with `yarn start` (use `CTRL-C` to exit).
```

Comment:
Maybe we should have yarn start run the storybook rather and remove the storybook script in package.json.

Reply:
Applied changes!